### PR TITLE
Test addition: Breadcrumbs and ManageSelector

### PR DESCRIPTION
**Pull request by nmirasch**

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @bsremac some test added related with manageSelector and breadcrumbs generation. Could you take a look? Thanks!

**Review by cristianonicolai — APPROVED** [review]

File: jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/menu/ManageSelectorImplTest.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        verifyInitSelectorBasicViewInteractions();
+        verify(view,
+               times(2)).addOption(anyString(),
+                                   anyString(),
+                                   anyBoolean());
+        verifyNoMoreInteractions(view);
+    }
+
+    private void verifyInitSelectorBasicViewInteractions() {
+        verify(view).removeAllOptions();
```
Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/menu/ManageSelectorImplTest.java]: I would keep this verification outside of this method.

File: jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/menu/ManageSelectorImplTest.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        manageSelector.init();
+        verifyInitSelectorBasicViewInteractions();
+        verify(view).addOption(Constants.INSTANCE.Process_Definitions(),
+                               PerspectiveIds.PROCESS_DEFINITIONS,
+                               true);
+        verify(view).addOption(Constants.INSTANCE.Process_Instances(),
+                               PerspectiveIds.PROCESS_INSTANCES,
+                               false);
+        verify(view).addOption(Constants.INSTANCE.Tasks(),
+                               PerspectiveIds.TASKS_ADMIN,
```
Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/menu/ManageSelectorImplTest.java:94]: @cristianonicolai @nmirasch One unrelated question. 
Do we want to continue having the perspective Id named TASKS_ADMIN?

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListPresenterTest.java [hunk 3]
```diff
@@ -190,4 +202,56 @@ public void testOnRuntimeDataServiceError() {
...
                times(2)).hideBusyIndicator();
     }
+
+    @Test
+    public void testListBreadcrumbCreation() {
+        presenter.createListBreadcrumb();
+        ArgumentCaptor<Command> captureCommand = ArgumentCaptor.forClass(Command.class);
+        verify(breadcrumbs).clearBreadcrumbs(PERSPECTIVE_ID);
+        verify(breadcrumbs).addBreadCrumb(eq(PERSPECTIVE_ID),
+                                          eq(org.jbpm.workbench.common.client.resources.i18n.Constants.INSTANCE.Home()),
```
Comment (bsremac) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/definition/list/ProcessDefinitionListPresenterTest.java]: How about making this line shorter by importing the class?

**Review by bsremac — COMMENTED** [review]
Looks good Nues. Thanks.
Just a couple of readability comments

File: jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/menu/ManageSelectorImplTest.java (hunk 2 above)
Comment (cristianonicolai) [reply jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/menu/ManageSelectorImplTest.java:94]: @bsremac I think we could rename that on master to the names are closer to the screen name.

**Comment by nmirasch** [comment]
@bsremac added commented changes about readability.

**Review by bsremac — APPROVED** [review]

**Pull request merged**